### DISCUSSION TITLE
Questions about ACE Medical

### DISCUSSION THREAD

**Discussion by oksmantv** [acemod/ACE3#8404] [Q&A] [answered]
Hello ACE team! I'm not sure if this is the right place to ask questions or not, but if it is, I'd like to start by saying thanks for making Arma 3 what it is today! This game wouldn't be the same without your efforts! :)

My question is regarding ACE Medical, my unit has just switched from the old system to the new system and we find ourselves using too much time on casualties for our liking and I was gonna check in and see if there's any ways to speed things up on our end.

We've adjusted the settings as best we can to reduce the complexity & time cost by removing the need for morphine & epinephrine, we only use bandages, IVs and PAKs as that is the simplest form we could extract out of the settings.

**Question 1:** Is there a way to make bandaging quicker? or to require less of it? We used to only require 1-3 bandages to heal a wound and now it can take up to 4-7. We're using elastic bandages only and we're not using advanced medical / bandages.

**Question 2:** Is there a way to shorten the PAK time? I see the coefficient setting and I tried reducing it to 0 but it didn't have an effect. Does this mean it's impossible to shorten the time of the PAK to say 5 seconds? Currently as I understand is, that it's only possibly to have a minimum of 10 seconds?

Thanks for your time & answers in advance! You're doing great, I hope we'll continue to see your efforts in this franchise for years to come!

Regards Oksman

**Comment by severgun** [comment, marked as answer]
> Question 1: Is there a way to make bandaging quicker? or to require less of it? We used to only require 1-3 bandages to heal a wound and now it can take up to 4-7. We're using elastic bandages only and we're not using advanced medical / bandages.

Change config to fixed value or write your own function to calc time.
https://github.com/acemod/ACE3/blob/76676eee462cb0bbe400a482561c148d8652b550/addons/medical_treatment/ACE_Medical_Treatment_Actions.hpp#L18

> Question 2:

Same
https://github.com/acemod/ACE3/blob/76676eee462cb0bbe400a482561c148d8652b550/addons/medical_treatment/ACE_Medical_Treatment_Actions.hpp#L304

**Reply by oksmantv** [reply]
Thanks a lot for such a quick response! 😄 

What would be the best way of applying this without repacking the addon? Since I'm doing this for an online group, if I have to repack the mod, I'd have to reupload it to get it to the members. But I'm guessing that's not really the best way to do it, and of course, is it something the ACE team might not allow? What other solutions can I do to apply this to say, mission files instead rather than repacking the addon?

Thanks a lot once again, I wish I knew exactly what you meant with your response but I'm left hanging a bit because of my limited technical skills with addons & configs. Any help will be much appreciated!

Regards Oksman

**Reply by veteran29** [reply]
You can write your own config patch addon, as you want more casual gameplay you might be interested in using (shameless self-advertisement): <https://github.com/ArmaForces/Medical>, If not you can see how to do the changes by looking into this addon.

**Reply by severgun** [reply]
> But I'm guessing that's not really the best way to do it, and of course, is it something the ACE team might not allow?

1) As @veteran29 said you can add config patch to separate addon. And keep ACE pbos native.
2) > The project is entirely open-source and all contributions are welcome. Feel free to maintain your own custom version, so long as the changes you make are open to the public in accordance with the GNU General Public License (GPLv2).
